### DISCUSSION TITLE
[vcpkg] Manifest Files

### DISCUSSION THREAD

**Discussion by strega-nil** [microsoft/vcpkg#17819] [Ideas]
This is the tracking issue for manifest files!

- [x] Add JSON parser: #10521 
- [x] Get the manifest specification merged: #11203
- [x] Initial demo support for manifest files: [strega-nil/vcpkg/manifest]
- [x] Full support for manifest files, behind a feature flag #11757 
- [ ] Switch all CONTROL files to manifest files
- [ ] Remove the feature flag

[strega-nil/vcpkg/manifest]: /strega-nil/vcpkg/tree/manifest

**Comment by linquize** [comment]
You change CONTROL file to JSON format

**Comment by linquize** [comment]
Will support of `CONTROL` files be eventually removed?

**Comment by strega-nil** [comment]
@linquize probably at some point? but it's not in the plans yet.
